src/sentry_value.c

```diff
@@ -1036,8 +1038,38 @@ sentry_value_new_breadcrumb(const char *type, const char *message)
     return rv;
 }
 
-void
-sentry_event_value_add_stacktrace(sentry_value_t event, void **ips, size_t len)
+sentry_value_t
+sentry_value_new_exception(const char *type, const char *value)
+{
+    sentry_value_t exc = sentry_value_new_object();
+    sentry_value_set_by_key(exc, "type", sentry_value_new_string(type));
+    sentry_value_set_by_key(exc, "value", sentry_value_new_string(value));
+    return exc;
+}
+
+sentry_value_t
+sentry_value_new_thread(uint64_t id, const char *name)
+{
+    sentry_value_t thread = sentry_value_new_object();
+
+    // TODO: would be nice to be able to actually use a `u64` as value.
```

Comment:
Unfortunately, this is not really a TODO. You have no choice.

While we [do support integers](https://github.com/getsentry/relay/blob/a1e81e59dfcbc362187dd0d5c04b1be66386d8e3/relay-general/src/protocol/thread.rs#L8-L17), it is not safe to use numbers larger than `2^53` in JSON. Hence, you'll have to stick with the string.

This is something we'll eventually even type out in Relay.